recordium/config.py

```diff
@@ -52,8 +52,8 @@ def __getattr__(self, key):
         return self.data.get(key, self._config_options[key])
 
     def __setattr__(self, key, value):
-        if key in self._config_options.keys():
-            if self.__getattr__(key) != value:
+        if key in self._config_options:
+            if self.data.get(key, self._config_options[key]) != value:
```

Comment:
What is the concept behind "not storing if the value is different of what the config has but the same than the default"?

Why not just "store if the value is different of what is already stored in the config"?